client/my-sites/marketing/index.js

```diff
@@ -26,6 +29,8 @@ export default function () {
 		'/sharing',
 		'/sharing/buttons',
 		'/marketing/business-tools',
+		'/marketing/do-it-for-me/landing',
```

Comment:
Maybe we can wrap these in a `config.isEnabled( 'difm-lite-pilot' )` as well?

Reply:
Thanks! nice catch :)